Improve Homebrew Portable Ruby messaging (#7476)
Homebrew/brew: docs/Homebrew-on-Linux.md

Homebrew can run on 32-bit ARM (Raspberry Pi and others) and 64-bit ARM (AArch64), but no binary packages (bottles) are available. Support for ARM is on a best-effort basis. Pull requests are welcome to improve the experience on ARM platforms.

> You may need to install your own Ruby using a e.g. PPA for a system package manager as in future we will no longer distribute a Homebrew Portable Ruby for ARM.

Reviewer: ```suggestion
You may need to install your own Ruby using e.g. a PPA for a system package manager as in the future we will no longer distribute a Homebrew Portable Ruby for ARM.
```

Reviewer 2: ```suggestion
You may need to install your own Ruby using a your system package manager, a PPA, or `rbenv/ruby-build` as in the future we will no longer distribute a Homebrew Portable Ruby for ARM.
```

Reviewer: `using a your` ➡️ `using your`
